src/components/BiobankExplorerContainer.vue

```diff
@@ -339,4 +339,8 @@ export default {
 .collection-cart > div:last-child {
   border:none !important;
 }
+
+.align-right {
```

Comment:
there is a bootstrap utility class for this

https://getbootstrap.com/docs/4.6/utilities/text/